dotnet/src/Planners/Planners.Core/Sequential/SKContextSequentialPlannerExtensions.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.Memory;
+using Microsoft.SemanticKernel.Planners.Sequential;
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using NS of SKContext
+namespace Microsoft.SemanticKernel.Orchestration;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// Provides extension methods for the <see cref="SKContext"/> class to work with sequential planners.
+/// </summary>
+public static class SKContextSequentialPlannerExtensions
```

Comment:
Will probably move this later to something that can be shared. Extensions|Utils folder in this project alongside Action|Sequential|Stepwise. Mostly a note for myself and a poll for preferences|ideas you might have on naming/placement.

Reply:
Making these shared utilities for all planners to use sounds great. I have a slight preference towards keeping this as an extension class in an adjacent Extensions folder.